src/MiniJuvix/Translation/AbstractToMicroJuvix.hs

```diff
@@ -14,19 +15,35 @@ import MiniJuvix.Syntax.MicroJuvix.MicroJuvixResult
 import MiniJuvix.Syntax.Universe
 import MiniJuvix.Syntax.Usage qualified as A
 
+newtype TranslationState = TranslationState
+  { -- | Top modules are supposed to be included at most once.
+    _stateIncluded :: HashSet A.TopModuleName
```

Comment:
_stateIncluded -> _translationStateModuleNames or similar

Reply:
solved in bc43ff62c88bcee81a746b77c460ffd879a41261